packages/clay-table/src/ClayTable.js

```diff
@@ -204,6 +240,15 @@ ClayTable.STATE = {
 	 */
 	showActionsMenu: Config.bool().value(false),
 
+	/**
+	 * Flag to indicate if the checkbox is show or not when selectable is true.
+	 * @default true
+	 * @instance
+	 * @memberof ClayTable
+	 * @type {?bool}
+	 */
+	showCheckbox: Config.bool().value(true),
```

Comment:
hey @carloslancha I'm putting the `true` value as default to keep the same behavior as before and not cause some breaking change. In this way to hide `showCheckbox`, just set true `selectable` and `showCheckbox` to false.

Reply:
Hey @matuzalemsteles if `selectable` default value is `false` I'd go with `showCheckbox` to false too. To avoid problems in portal we can set this to `true` by default in the taglib, so here we can have a more "logic" definition. What do you think?

Reply:
I do not know if this would cause some strange behavior in the Portal, I'm assuming that when selectable is false the checkbox will not be shown and when selectable is true it will be shown, this keeps the default as it was before, so will have no problems.

I think if we only keep in taglib as true, in the rehydration of the elements the checkbox may disappear.

makes sense?

Reply:
`I think if we only keep in taglib as true, in the rehydration of the elements the checkbox may disappear`.

Nope, the Tag is in charge of create both server and client component instances with same values.